### PR TITLE
change password scenario

### PR DESCRIPTION
**Pull request by chrisleewilcox**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions,_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: wdio/step-definitions/change-password.steps.js [hunk 1]
```diff
@@ -27,4 +27,5 @@ When(/^I tap Reset password/, async () => {
 Then(/^Creating password is displayed/, async () => {
   await driver.pause(1000);
   await WalletMainScreen.isTokenTextVisible('Creating password...');
```
Comment (cortisiko) [inline wdio/step-definitions/change-password.steps.js:29]: maybe this was a typo?

**Review by cortisiko — APPROVED** [review]
just left one comment. Outside of that, the code looks good.

File: wdio/step-definitions/change-password.steps.js (hunk 1 above)
Comment (chrisleewilcox) [reply wdio/step-definitions/change-password.steps.js:29]: I saw this existing feature and used it to confirm the text that is displayed after changing password.  I'll add another function that does the same in relation to confirming change password.

Comment (cortisiko) [reply wdio/step-definitions/change-password.steps.js:29]: ah gotcha! Sounds good.

**Comment by chrisleewilcox** [comment]
detox
https://app.bitrise.io/build/4ffa2b71-8eeb-4565-8191-34f0e2ce6567

**Comment by chrisleewilcox** [comment]
browserstack
https://app-automate.browserstack.com/dashboard/v2/builds/af352c3cb4fc61986127b52d3b21c37a2213b608/sessions/a625ce1d5c309a33ba777c3337aa8533b73e9d55

**Pull request merged**